Concept ACK, I think it's better to log something. Could add logic to log only every so many %, but this is a good start.

It does fail to build here, might need rebase for recent changes:
```
/home/orion/projects/bitcoin/bitcoin/src/validation.cpp: In function ‘bool ProcessNewBlockHeaders(const std::vector<CBlockHeader>&, CValidationState&, const CChainParams&, const CBlockIndex**, CBlockHeader*)’:
/home/orion/projects/bitcoin/bitcoin/src/validation.cpp:3332:13: error: ‘IsInitialBlockDownload’ was not declared in this scope
         if (IsInitialBlockDownload() && ppindex && *ppindex) {
             ^~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/validation.cpp
+++ b/src/validation.cpp
@@ -3327,6 +3327,12 @@ bool ProcessNewBlockHeaders(const std::vector<CBlockHeader>& headers, CValidatio
         }
     }
     NotifyHeaderTip();
+    {
+        LOCK(cs_main);
+        if (::ChainstateActive().IsInitialBlockDownload() && ppindex && *ppindex) {
+            LogPrintf("Synchronizing blockheaders, height: %d (~%.2f%%)\n", (*ppindex)->nHeight, 100.0/((*ppindex)->nHeight+(GetAdjustedTime() - (*ppindex)->GetBlockTime()) / Params().GetConsensus().nPowTargetSpacing) * (*ppindex)->nHeight);
+        }
+    }
     return true;
 }
 

